src/System.Private.CoreLib/src/System/Diagnostics/Contracts/Contracts.cs

```diff
@@ -629,7 +629,7 @@ public static bool ForAll(int fromInclusive, int toExclusive, Predicate<int> pre
 #if INSIDE_CLR
                 throw new ArgumentException(SR.Argument_ToExclusiveLessThanFromExclusive);
 #else
-                throw new ArgumentException("fromInclusive must be less than or equal to toExclusive.");
+                throw new ArgumentException($"{nameof(fromInclusive)} must be less than or equal to {nameof(toExclusive)}.");
 #endif
```

Comment:
Agree.

This ifdef is wrong. It was not adjusted correctly as it was copied from CoreCLR. The ifdef should be deleted: the code under `INSIDE_CLR` that uses the resource string should be kept, and the else part deleted.